prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -511,6 +511,56 @@ ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_pee
 	return 0;
 }
 
+ssize_t rxr_pkt_trigger_handshake(struct rxr_ep *ep,
```

Comment:
Ad function header here. Also ```rxr_pkt_wait_handshake``` can be sampled by using the new function ```rxr_pkt_trigger_handshake```.

Reply:
The implementation of `rxr_pkt_wait_handshake` is already included in the pull request https://github.com/ofiwg/libfabric/pull/6144. Shall I open another PR for reimplementation of `rxr_pkt_wait_handshake`?

Reply:
I think re-implement it when you introduce ```rxr_pkt_trigger_handshake``` is ok. You just need to document it in the commit message

Reply:
`rxr_pkt_wait_handshake` is re-implemented. Resolving.